hail_search/queries/base.py

```diff
@@ -237,7 +237,8 @@ def __init__(self, sample_data, sort=XPOS, sort_metadata=None, num_results=100,
         self._has_secondary_annotations = False
         self._is_multi_data_type_comp_het = False
         self.max_unaffected_samples = None
-        self._load_table_kwargs = {'_n_partitions': min(MAX_PARTITIONS, (os.cpu_count() or 2)-1)}
+        self._n_partitions = min(MAX_PARTITIONS, (os.cpu_count() or 2)-1)
+        self._load_table_kwargs = {'_n_partitions': self._n_partitions}
```

Comment:
Why did you keep `self._load_table_kwargs` here? To me it seems superfluous since it's referencing the same thing as ` self._n_partitions`.

Reply:
So when any table is read in `_load_table_kwargs` is passed into the read function. Under certain search conditions and data types certain searches use different kwargs, so essentially this is setting the default value and then it may be updated elsewhere. It would be impossible to replicate that behavior cleanly without leaving this as is